react_app/src/Clock.js

```diff
@@ -0,0 +1,64 @@
+import React,{Component} from 'react';
+
+const clockTile = {
+  color: 'white',
+  textAlign:'center'
+}
+const clockSpan = {
+  padding : '4px'
+}
+let runner;
+class Clock extends Component{
+  constructor(props) {
+    super(props);
+    this.state ={
+      'time' : this.getCurrentTime()
+    }
+  }
+  getCurrentTime = () =>{
+    const locale = this.props.locale ? this.props.locale : [];
+    const hour12 = (this.props.hour12 === false) ? false : true;
+    let hour,minute,second;
```

Comment:
Looks to me like hour will always be 2 digit, why not go ahead and create it with that info and have the case 'hh'  break without assigning